Source/Scene/Scene.js

```diff
@@ -505,6 +505,12 @@ define([
          */
         this.copyGlobeDepth = false;
 
+        this.fog = {
+            enabled : true,
+            density : 0.001,
```

Comment:
I haven't tried tweaking this yet, but I'm still not sure how a user will have intuitive control of this.  For example, if I am writing a geoportal for a city, I might want fog to start 1 km out and then be complete fog in 1 km.  How does this map to this `density`?  If we can't expose more intuitive parameters, we at least need to provide doc on set to determine this.